src/Analyzers/Analyzers/src/StartupAnalyzer.Diagnostics.cs

```diff
@@ -53,5 +38,15 @@ static Diagnostics()
             DiagnosticSeverity.Warning,
             isEnabledByDefault: true,
             helpLinkUri: "https://aka.ms/AA64fv1");
+
+        public static readonly ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics = ImmutableArray.Create<DiagnosticDescriptor>(new[]
```

Comment:
Static fields are initialized in the order in which they appear, but you can reference them before they are initialized. Cool new thing I found out today.